src/Bicep.Decompiler/UniqueNamingResolver.cs

```diff
@@ -16,6 +16,15 @@ public class UniqueNamingResolver : INamingResolver
 
         private readonly Dictionary<string, string> assignedResourceNames = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
 
+        private static string GetNamingSuffix(NameType nameType)
+            => nameType switch {
+                NameType.Output => "out",
```

Comment:
>out [](start = 36, length = 3)

Should we include an underscore to separate it from the "main" part of the name?